dts/bindings/pinctrl/atmel,sam0-pinctrl.yaml

```diff
@@ -105,6 +95,17 @@ child-binding:
   child-binding:
     description: |
       The grandchild nodes group pins that share the same pin configuration.
+
+    include:
+      - name: pincfg-node.yaml
+        property-allowlist:
+          - bias-pull-up
+          - bias-pull-down
+          - drive-open-drain
```

Comment:
`drive-open-drain` was not included in the existing allow list?

Reply:
thanks for catching, fixed